io/io/src/TFile.cxx

```diff
@@ -3613,6 +3611,7 @@ void TFile::ReadStreamerInfo()
       }
    }
 
+   std::vector<Int_t> si_uids;
```

Comment:
Why is this not `RUidColl`?
```suggestion
   RUidColl si_uids;
```

Reply:
oversight.

Reply:
I think I will leave it as is.  It feel clearer with the concrete type here and the match is guaranteed by the `std::move` used later on.

Reply:
Then why introduce the public type alias at all?

Reply:
To shorten the header, to self document the header.